src/path-rewriter.ts

```diff
@@ -42,13 +42,9 @@ export function createPathRewriter(rewriteConfig) {
 function isValidRewriteConfig(rewriteConfig) {
   if (typeof rewriteConfig === 'function') {
     return true;
-  } else if (_.isPlainObject(rewriteConfig) && Object.keys(rewriteConfig).length !== 0) {
-    return true;
-  } else if (
-    rewriteConfig === undefined ||
-    rewriteConfig === null ||
-    _.isEqual(rewriteConfig, {})
```

Comment:
`_.isEqual(rewriteConfig, {})` check is now missing in the refactored code. Think the vanilla version would be: `JSON.stringify(rewriteConfig) === '{}'`.

Reply:
Comparing to empty object means it should be plain one, right? There is plain object check above. I moved Object.keys length check to return value instead.

Reply:
Ah, you're right. Missed that part. 👌